src/validation.h

```diff
@@ -686,8 +714,7 @@ CChain& ChainActive();
 /** @returns the global block index map. */
 BlockMap& BlockIndex();
 
-/** Global variable that points to the coins database (protected by cs_main) */
-extern std::unique_ptr<CCoinsViewDB> pcoinsdbview;
+extern std::unique_ptr<CChainState> g_chainstate;
```

Comment:
In commit "refactor: have CCoins* data managed under CChainState" (eed4e62ad6b162ff0d3e2b87eb8762a689678bca)

Can you add a comment here about when it's appropriate to access `g_chainstate` variable directly, as opposed to calling `ChainstateActive()`? I see code in `init.cpp` is using both, but mostly the former, which is different than what I'd expect. I'd expect init code to just create a CChainState object and assign it it to `g_chainstate` once rather than accessing the `g_chainstate` global repeatedly.